kf_lib_data_ingest/config.py

```diff
@@ -11,9 +11,7 @@
 TARGET_URL_KEY = 'target_url'
 DEFAULT_TARGET_URL = 'http://localhost:5000'
 KIDSFIRST_DATASERVICE_PROD_URL = 'http://kf-api-dataservice.kidsfirstdrc.org'
```

Comment:
It's weird that KIDSFIRST_DATASERVICE_PROD_URL isn't in the same place as TARGET_SERVICE_CONFIG_PATH

Reply:
👍 Good point

Reply:
Actually, I think this is a left over thing from when things were a little different with null processing. Its not used anywhere in the source code anymore. The target_url is used when fetching target service schemas. 

KIDSFIRST_DATASERVICE_PROD_URL is only used in handle_nulls tests so I can move it to conftest. However, since we mock everything out, it probably doesn't even matter what URL we use for testing